book.rb

```diff
@@ -0,0 +1,15 @@
+# Book class
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
```

Comment:
Getters and Setters for `rentals` should be present, Kindly add the `rentals` to attr_accessor. 👍